traffic_portal/app/src/assets/css/angular-ui-tree_2.22.6.css

```diff
@@ -0,0 +1,146 @@
+.angular-ui-tree-empty {
+    border:1px dashed #bbb;
+    min-height:100px;
+    background-color:#e5e5e5;
+    background-image:-webkit-linear-gradient(45deg,#fff 25%,transparent 0,transparent 75%,#fff 0,#fff),-webkit-linear-gradient(45deg,#fff 25%,transparent 0,transparent 75%,#fff 0,#fff);
+    background-image:linear-gradient(45deg,#fff 25%,transparent 0,transparent 75%,#fff 0,#fff),linear-gradient(45deg,#fff 25%,transparent 0,transparent 75%,#fff 0,#fff);
+    background-size:60px 60px;
+    background-position:0 0,30px 30px;
+    pointer-events:none
+}
+.angular-ui-tree-nodes {
+    position:relative;
+    margin:0;
+    padding:0;
+    list-style:none
+}
+
+.angular-ui-tree-nodes .angular-ui-tree-nodes {
+    padding-left:30px;
+}
+
+.angular-ui-tree-node, .angular-ui-tree-placeholder {
+    position:relative;
+    margin:0;
+    padding:0;
+    min-height:20px;
+    line-height:20px
+}
+
+.angular-ui-tree-hidden {
+    display:none
+}
+
+.angular-ui-tree-placeholder {
+    margin:5px 0;
+    padding:0;
+    min-height:30px
+}
+
+.angular-ui-tree-handle {
+    cursor:move;
+    text-decoration:none;
+    font-weight:700;
+    box-sizing:border-box;
+    min-height:20px;
+    line-height:20px
+}
+
+.angular-ui-tree-drag {
+    position:absolute;
+    pointer-events:none;
+    z-index:999;
+    opacity:.8
+}
+
+.btn {
+    margin-right: 8px;
+    margin-bottom: 0;
+}
+
+.angular-ui-tree-handle {
+    background: #fff0f0;
+    border: 1px solid #dae2ea;
+    color: #7c9eb2;
+    padding: 10px 10px;
+}
+
+.angular-ui-tree-handle:hover {
+    color: #155724 !important;
+    background-color: #D4EDD7 !important;
+    border-color: #c3e6cb !important;
+}
+
+.angular-ui-tree-placeholder {
+    background: #f0f9ff;
+    border: 2px dashed #bed2db;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+tr.angular-ui-tree-empty {
+    height: 100px
+}
+
+.group-title {
+    background-color: #687074 !important;
+    color: #FFF !important;
+}
+
+.tree-node {
+    border: 1px solid #dae2ea;
+    background: #f8faff;
+    color: #7c9eb2;
+}
+
+.nodrop {
+    background-color: #f2dede;
+}
+
+.tree-node-content {
+    height: 45px;
+    text-align: center;
+    font-size: 14px;
+    margin-bottom: 10px;
+}
+
+.tree-handle {
+    padding: 10px;
+    background: #428bca;
+    color: #FFF;
+    margin-right: 10px;
+}
+
+.angular-ui-tree-handle:hover {}
+
+.angular-ui-tree-placeholder {
+    background: #f0f9ff;
+    border: 2px dashed #bed2db;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+.node circle {
+    fill: #999;
+}
+
+.node text {
+    font: 10px sans-serif;
+}
+
+.node--internal circle {
+    fill: #555;
+}
+
+.node--internal text {
+    text-shadow: 0 1px 0 #fff, 0 -1px 0 #fff, 1px 0 0 #fff, -1px 0 0 #fff;
+}
+
+.link {
+    fill: none;
+    stroke: #555;
+    stroke-opacity: 0.4;
+    stroke-width: 1.5px;
+}
```

Comment:
This is library code? Or you wrote it yourself?

It doesn't have a license header so I'm assuming the former. Can it be minified?

Reply:
minified

Reply:
The file still needs a license after minification.

Reply:
not if it's library code, because then it should be covered by the dependency licenses.

Reply:
I see. Left a comment [on dependency licenses](https://github.com/apache/trafficcontrol/pull/4633#discussion_r413492085) because the patterns need to be adjusted to avoid Weasel warnings.

Reply:
PR submitted to fix this